holdings_maintenance/pds4/pds4checksums.py

```
@@ -0,0 +1,859 @@
+
+    finally:
+        logger.close()
+        return success
+
+################################################################################
+
+def move_old_checksums(check_path, logger=None):
+    """Appends a version number to an existing checksum file and moves it to
+    the associated log directory."""
+
+    if not os.path.exists(check_path): return
```

Flake8 (7.2.0)

[error] multiple statements on one line (colon)

(E701)